docs/cosmwasm.md

```diff
@@ -0,0 +1,57 @@
+# Cosmwasm
+
+## Cosmwasm Built-in capabilities 
+   - [Built-in capabilities](https://github.com/CosmWasm/cosmwasm/blob/main/docs/CAPABILITIES-BUILT-IN.md) - iterator, staking, stargate, cosmwasm_1_1, cosmwasm_1_2
+   - Custom capability of umee chain 
+     - umee
+
+## Allowed native module queries 
+
+- [ugov](https:github.com/umee-network/umee/blob/main/proto/umee/ugov/v1/query.proto)
+- [leverage](https:github.com/umee-network/umee/blob/main/proto/umee/leverage/v1/query.proto)
+- [oracle](https:github.com/umee-network/umee/blob/main/proto/umee/oracle/v1/query.proto)
+- [uibc](https:github.com/umee-network/umee/blob/main/proto/umee/uibc/v1/quota.proto)
+
+```json
+// queries for anything available from the umee native modules (ugov,leverage,oracle.uibc)
```

Comment:
`//` is not a JSON comment (in fact, JSON doesn't support comments). Let's move it outside of the block.

Reply:
ok